docs/source/python/api/compute.rst

```diff
@@ -127,6 +129,18 @@ variants which detect domain errors where appropriate.
    tan
    tan_checked
 
+Rounding
```

Comment:
Did you mean to add rounding functions twice in this file?

Reply:
Not at all.